.k8s/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
I think some of this could be simplified or removed. For instance, there are handlers in here for supporting a full name override flag. Also, we don't really need to append `-standard-web` to the deployment name.

Reply:
Better?